x-pack/platform/plugins/private/reporting/server/saved_objects/scheduled_report/model_versions.ts

```diff
@@ -42,4 +43,20 @@ export const scheduledReportModelVersions: SavedObjectsModelVersionMap = {
       create: rawScheduledReportSchemaV4,
     },
   },
+  '5': {
+    changes: [
+      {
+        type: 'mappings_addition',
+        addedMappings: {
+          title: {
+            type: 'text',
+          },
+        },
+      },
+    ],
+    schemas: {
+      forwardCompatibility: rawScheduledReportSchemaV5.extends({}, { unknowns: 'ignore' }),
```

Comment:
I’ve never found myself having to introduce a new SO version that only changes the mappings, but not the schema. SO docs showed an example without schemas at all, but CI errored because of missing `forwardCompatibility`, so I added it. But using the schema v4 in model version 5 didn't feel right, so I advanced the schema version to 5, just re-exporting all from schema 4. I don't know if this is the best way to do it, so any suggestion is appreciated